aws/res/bootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euo pipefail; IFS=$'\n\t'
+
+#
+# 기본 유틸리티들 설치
+#
```

Comment:
미니멀 AMI인데도 안쓰는 postfix가 들어있어서 지우면 좋을거같아요

```suggestion
#
yum autoremove -y postfix
```